plugins/modules/junos_package.py

```diff
@@ -52,6 +52,12 @@
       to the remote device prior to installing.
     type: bool
     default: no
+  unlink:
+    description:
+    - The I(unlink) argument is responsible for instructing the remote device to
+    remove the package after installation.
+    type: bool
+    default: no
```

Comment:
```suggestion
    default: false
```

Reply:
We're trying to adhere to https://github.com/ansible-collections/junipernetworks.junos/issues/376.

Reply:
I've changed this to false